integration/spark-common/src/main/scala/org/apache/spark/sql/catalyst/CarbonDDLSqlParser.scala

```diff
@@ -1044,13 +1177,13 @@ abstract class CarbonDDLSqlParser extends AbstractCarbonSparkSQLParser {
    * Matching the decimal(10,0) data type and returning the same.
    */
   private lazy val decimalType =
-  DECIMAL ~ (("(" ~> numericLit <~ ",") ~ (numericLit <~ ")")).? ^^ {
-    case decimal ~ precisionAndScale => if (precisionAndScale.isDefined) {
-      s"decimal(${ precisionAndScale.get._1 }, ${ precisionAndScale.get._2 })"
-    } else {
-      s"decimal(10,0)"
+    DECIMAL ~ (("(" ~> numericLit <~ ",") ~ (numericLit <~ ")")).? ^^ {
```

Comment:
what is the change here?

Reply:
these changes are done by mistake while formatting, i will revert these unwanted changes